src/foam/box/HTTPBox.js

```diff
@@ -26,8 +26,9 @@ foam.CLASS({
   ],
 
   imports: [
+    'generator? as ctxGenerator',
```

Comment:
> No you would only have to subclass the two you want to replace (MessagePortService and MessagePortBox) and then use the context to replace which ones get created throughout the rest of the box system.

I don't see how I can use the context to make `(Raw)Socket(Box|Service)` and `(Raw)WebSocket(Box|Service)` use this serialization strategy. I would need to subclass them too if the strategy isn't customizable.

> every knob we add the framework adds complexity and overhead for all other organizations that want to use it.

In my view, adding `JSONParserStringifier` does add a second way of doing serialization (and, in that sense, complexity and/or cognitive overhead). However, making serialization customizable only adds complexity to those who care to customize it. Any users of the framework happy with the default `parser` and `serializer` don't even need to know that they can be customized. I think a case can be made for *not* including `JSONParserStringifier` in the framework more so than *not* allow users to choose a different strategy.

> If you turn strict:true on in the existing serializers it is supposed to output proper json that you could use JSON.parse on, and this wouldn't require the intermediate objectify() step, which will hoepfully create less garbage.

Good point. Perhaps `strict: true` on the stringifier side with a native `JSON.parse` on the parser side is the sweet spot here. The use case that uncovered performance issues showed them on the parsing side (though I haven't measured the stringifier side). Do you think this approach would work?